router/router.go

```diff
@@ -2447,11 +2441,11 @@ func (rt *HandleT) HandleOAuthDestResponse(params *HandleDestOAuthRespParamsT) (
 			return http.StatusInternalServerError, trRespBody
 		}
 	}
-	// By default send the status code & response from transformed response directly
+	// By default, send the status code & response from transformed response directly
 	return trRespStatusCode, trRespBody
 }
 
-func (rt *HandleT) ExecDisableDestination(destinationJob types.DestinationJobT, workspaceId, destResBody, rudderAccountId string) (int, string) {
+func (rt *HandleT) ExecDisableDestination(destinationJob *types.DestinationJobT, workspaceId, destResBody, rudderAccountId string) (int, string) {
```

Comment:
we dont need a `types.DestinationJobT` but a `backendconfig.DestinationT`, correct?
```suggestion
func (rt *HandleT) ExecDisableDestination(destination backendconfig.DestinationT, workspaceId, destResBody, rudderAccountId string) (int, string) {
```